main/main.cpp

```diff
@@ -2457,9 +2469,35 @@ Error Main::setup2() {
 	/* Initialize Display Server */
 
 	{
+		if (display_driver.is_empty()) {
+			display_driver = GLOBAL_GET("display/display_server/driver");
+		}
+
```

Comment:
As a codestyle nitpick, benchmarking calls must be wrapping the scope, so this addition needs to be moved below `OS::get_singleton()->benchmark_begin_measure("Servers", "Display");`